src/controllers/userController.js

```diff
@@ -0,0 +1,70 @@
+import userService from '../services/userService';
+import { jwtSignUser } from '../utils/index';
+import { comparePassword } from '../helpers/comparePassword';
+import { logins } from '../models';
+/**
+ * @class UsersController
+ */
+class UserController {
+  /** Login User
+   * @static
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @returns {string} loginUsers
+   */
+  static async loginAUser(req, res) {
+    const { email, password } = req.body;
+    try {
+      const loggedUser = await userService.loginAUser(email);
+      if (!loggedUser) {
+        return res.status(404).json({
+          error: 'Email does not exit',
```

Comment:
why not tell the user Email or password does not exist

Reply:
ok, thanks